subworkflows/local/bam_variant_calling_germline_all/main.nf

```diff
@@ -132,7 +137,7 @@ workflow BAM_VARIANT_CALLING_GERMLINE_ALL {
 
         if (joint_germline) {
             BAM_JOINT_CALLING_GERMLINE_GATK(
-                BAM_VARIANT_CALLING_HAPLOTYPECALLER.out.genotype_intervals,
+                BAM_VARIANT_CALLING_HAPLOTYPECALLER.out.genotype_intervals_and_crams,
```

Comment:
I'd rather that be done on `dev` than here

Reply:
I'll undo, and instead make a github-issue for it. (I found it very confusion when working with this that the `genotype_intervals` wasn't just intervals (bed-files) but also cram-files.)